versioned_hdf5/tests/test_subchunk_map.py

```diff
@@ -10,16 +10,22 @@
 from hypothesis.extra import numpy as stnp
 from numpy.testing import assert_array_equal
 
-from ..subchunk_map import as_subchunk_map
+from ..subchunk_map import (
+    DROP_AXIS,
+    EverythingMapper,
+    SliceMapper,
+    as_subchunk_map,
+    index_chunk_mappers,
+)
 
 max_examples = 10_000
 
 
 def non_negative_step_slices(size: int):
-    start = st.one_of(st.integers(-size - 1, size + 1), st.none())
-    stop = st.one_of(st.integers(-size - 1, size + 1), st.none())
+    start = st.one_of(st.none(), st.integers(-size - 1, size + 1))
+    stop = st.one_of(st.none(), st.integers(-size - 1, size + 1))
     # only non-negative steps (or None) are allowed
-    step = st.one_of(st.integers(1, size + 1), st.none())
+    step = st.one_of(st.none(), st.integers(1, size + 1))
```

Comment:
This is a nit following hypothesis best practices from https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.one_of

In practice this doesn't matter due to the very high cardinality of our tests and the fact that all slices are reduced by ndindex very early on.